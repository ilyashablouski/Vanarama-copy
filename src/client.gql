extend type Query {
  selectedOrderUuid: ID
  selectedDerivativeId: ID
}
