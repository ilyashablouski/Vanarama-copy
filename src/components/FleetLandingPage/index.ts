import FleetLandingPage from './FleetLandingPage';

export default FleetLandingPage;
