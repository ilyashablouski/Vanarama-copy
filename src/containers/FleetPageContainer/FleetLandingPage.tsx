import React from 'react';
import dynamic from 'next/dynamic';
import SchemaJSON from '@vanarama/uibook/lib/components/atoms/schema-json';
import { GetFleetLandingPage } from '../../../generated/GetFleetLandingPage';
import Head from '../../components/Head/Head';
import Skeleton from '../../components/Skeleton';

const HeroSection = dynamic(() => import('./sections/HeroSection'), {
  loading: () => <Skeleton count={5} />,
});

const LeadTextSection = dynamic(() => import('./sections/LeadTextSection'), {
  loading: () => <Skeleton count={5} />,
});

const TestimonialSection = dynamic(
  () => import('./sections/TestimonialSection'),
  {
    loading: () => <Skeleton count={5} />,
  },
);

const MediaFeatureSection = dynamic(
  () => import('./sections/MediaFeatureSection'),
  {
    loading: () => <Skeleton count={5} />,
  },
);

const BenefitsSection = dynamic(() => import('./sections/BenefitsSection'), {
  loading: () => <Skeleton count={5} />,
});

interface IFleetLandingPage {
  data: GetFleetLandingPage | undefined;
}

const FleetLandingPage = ({ data }: IFleetLandingPage) => {
  return (
    <>
      {data?.fleetLandingPage?.sections?.hero && (
        <HeroSection {...data?.fleetLandingPage?.sections?.hero} />
      )}
      {data?.fleetLandingPage?.sections?.leadText && (
        <LeadTextSection {...data?.fleetLandingPage?.sections?.leadText} />
      )}
      {data?.fleetLandingPage?.sections?.featured1 && (
        <TestimonialSection {...data?.fleetLandingPage?.sections?.featured1} />
      )}
      {data?.fleetLandingPage?.sections?.featured2 && (
        <MediaFeatureSection {...data?.fleetLandingPage?.sections?.featured2} />
      )}
      {data?.fleetLandingPage?.sections?.featured3 && (
        <MediaFeatureSection {...data?.fleetLandingPage?.sections?.featured3} />
      )}
      {data?.fleetLandingPage?.sections?.featured4 && (
        <MediaFeatureSection {...data?.fleetLandingPage?.sections?.featured4} />
      )}
      <hr className="-fullwidth" />
      {data?.fleetLandingPage?.sections?.tiles && (
        <BenefitsSection {...data?.fleetLandingPage?.sections?.tiles} />
      )}
      <SchemaJSON
        json={JSON.stringify(data?.fleetLandingPage?.metaData?.schema)}
      />
      {data?.fleetLandingPage.metaData && (
        <>
          <Head
            metaData={data?.fleetLandingPage.metaData}
            featuredImage={data?.fleetLandingPage.featuredImage}
          />
          <SchemaJSON
            json={JSON.stringify(data?.fleetLandingPage.metaData.schema)}
          />
        </>
      )}
    </>
  );
};

export default FleetLandingPage;
