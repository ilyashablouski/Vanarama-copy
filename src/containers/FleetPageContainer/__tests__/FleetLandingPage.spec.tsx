import React from 'react';
// @ts-ignore
import preloadAll from 'jest-next-dynamic';
import { screen, render, waitFor } from '@testing-library/react';
import { MockedProvider } from '@apollo/client/testing';
import FleetLandingPage from '../FleetLandingPage';

import { GenericPageQuery_genericPage as FleetPageData } from '../../../../generated/GenericPageQuery';

jest.mock('next/router', () => ({
  useRouter: () => ({
    asPath: '/',
  }),
}));

// ARRANGE
const DATA = {
  genericPage: {
    id: '1iXmVZNIGFolU38Jol4j08',
    sections: {
      featured1: {
        title:
          "Wherever You Are, However You Want To Work, We're There For You",
        titleTag: null,
        body:
          "The prospect of managing a mixed fleet of multiple vehicles, on varied financial contracts, is daunting...but we'll take that away. Our dedicated team, and the platforms available to you and us, mean that running your mixed fleet is hassle-free.",
        layout: ['Media Right'],
        testimonials: [
          {
            customerName: 'Peter Cheshire',
            summary:
              'Vanarama offered the cheapest prices on the vehicles and their service is fully integrated from telephone support to a state-of-the-art app, for Drivers and Fleet Managers.',
            rating: '5',
          },
        ],
      },
      featured2: {
        title: "What's In It For My Business?",
        titleTag: null,
        body:
          "### Every fleet is different: \nYour fleet needs are unique and that's why entrusting yours to a company you trust is important.\n\n### Best price for your business:\nChoosing a fleet management company that works with a network of funders means they can provide the very best prices on the market first time.\n\n### Compliance as standard: \nDuty of care is a key concern and goes hand-in-hand with staying compliant. Driver risk assessments and safety checks need to be carried out on time and to high standards, so work with a company that understands how to keep you organised.\n",
        image: null,
        layout: ['Media Left'],
      },
      featured3: {
        title: "What's In It For Me?",
        titleTag: null,
        body:
          "### Hassle-free fleet reporting:\nFrom MOT notifications to duty of care compliance tasks, keeping information at your fingertips is vital to effective fleet management.\n\n### Protect your fleet: \nHaving your fleet managed and insured through one channel makes life less complicated for you.\n\n### Cost-effective fleet maintenance: \nEasy access to all routine servicing and repairs, tyre replacement, MOTs and vehicle breakdown cover is true peace-of-mind.\n\n### Sale and lease back: \nRemoving vehicles from outright purchase and into a lease agreement allows you to grow your business faster by giving you the option to invest cash into your business rather than the vehicles.\n\n### Controlling costs: \nA robust reporting platform will allow you to look at maintenance budgets, total driver costs and whole-life fleet costings at a moment's notice.",
        image: {
          title: 'AudiQ30718 4 k5ojqt',
          file: {
            url:
              '//images.ctfassets.net/3xid768u5joa/2QbzKD75ObIHIeVMzNeCF/1455166780ba4eb095c965e278a63b69/AudiQ30718_4_k5ojqt.jpg',
          },
        },
        layout: ['Media Right'],
      },
      featured4: {
        title: "What's In It For The Drivers?",
        titleTag: null,
        body:
          "### Maintenance when it's needed: \nNo matter which maintenance package you choose, the drivers themselves can arrange bookings through the Driverhelpline application or by calling the support team.\n\n### Comprehensive vehicle management: \nHow great would it be if drivers knew that support was just a tap on an app away? They'll benefit from the same up-to-date reporting and service-on-demand functions as you. Drivers know they're being compliant, and you can rest easy.\n\n### Peace of mind and support:\nFor drivers, the fleet management relationship shouldn't be centred on vehicle delivery, they want support. And they'll have questions they need answered. That's when having a dedicated app filled with the information they need, and a dedicated support team really will pay for itself.",
        image: null,
        layout: ['Media Left'],
      },
      leadText: {
        heading: 'Why You Should Trust Your Fleet With Vanarama',
        titleTag: null,
        description:
          "The prospect of managing a mixed fleet of multiple vehicles, on varied financial contracts, is daunting...but we'll take that away. Our dedicated team, and the platforms available to you and us, mean that running your mixed fleet is hassle-free",
      },
      hero: {
        title: 'Flexible Fleet Management ',
        body: 'With __No Hidden Costs__',
        image: null,
      },
      tiles: {
        name: 'Not Quite Convinced? Also Included:',
        tiles: [
          {
            body: 'Pretium facilisi etiam pretium, cras interdum enim, nullam.',
            title: 'Price Protection',
            image: null,
          },
          {
            body: 'Pretium facilisi etiam pretium, cras interdum enim, nullam.',
            title: 'Customer Reviews',
            image: null,
          },
          {
            body: 'Pretium facilisi etiam pretium, cras interdum enim, nullam.',
            title: 'Quote Online',
            image: null,
          },
          {
            body: 'Pretium facilisi etiam pretium, cras interdum enim, nullam.',
            title: 'Confused About Leasing?',
            image: null,
          },
        ],
      },
    },
  } as FleetPageData,
};

describe('<FleetLandingPage />', () => {
  beforeAll(async () => {
    await preloadAll();
  });

  it('should render hero section correctly', async () => {
    // ACT
    render(
      <MockedProvider addTypename={false}>
        <FleetLandingPage genericPage={DATA.genericPage} />
      </MockedProvider>,
    );

    // ASSERT
    await waitFor(() => {
      expect(screen.getByText(`Flexible Fleet Management`)).toBeInTheDocument();
    });
    expect(screen.getByText('No Hidden Costs')).toBeVisible();
    expect(screen.getByTestId('fleet-request-call-back-form')).toBeVisible();
    expect(screen.getByTestId('fleet-call-back-form_full-name')).toBeVisible();
    expect(
      screen.getByTestId('fleet-call-back-form_company-name'),
    ).toBeVisible();
    expect(screen.getByTestId('fleet-call-back-form_fleet-size')).toBeVisible();
    expect(
      screen.getByTestId('fleet-call-back-form_email-address'),
    ).toBeVisible();
    expect(
      screen.getByTestId('fleet-call-back-form_phone-number'),
    ).toBeVisible();
    expect(screen.getByTestId('aboutTermsAndCons')).toBeVisible();
    expect(screen.getByTestId('aboutPrivacyPolicy')).toBeVisible();
    expect(screen.getByTestId('aboutConsent')).toBeVisible();
  });

  it('should render testimonial section correctly', async () => {
    // ACT
    render(
      <MockedProvider addTypename={false}>
        <FleetLandingPage genericPage={DATA.genericPage} />
      </MockedProvider>,
    );

    // ASSERT
    await waitFor(() => {
      expect(
        screen.getByText(
          `Vanarama offered the cheapest prices on the vehicles and their service is fully integrated from telephone support to a state-of-the-art app, for Drivers and Fleet Managers.`,
        ),
      ).toBeInTheDocument();
    });
    expect(
      screen.getByText(
        `Wherever You Are, However You Want To Work, We're There For You`,
      ),
    ).toBeVisible();
    expect(
      screen.getByTestId('fleet_testimonial-section_request-button'),
    ).toHaveTextContent("I'd Like A Callback");
  });

  it('should render side media sections according to layout type', async () => {
    // ACT
    render(
      <MockedProvider addTypename={false}>
        <FleetLandingPage genericPage={DATA.genericPage} />
      </MockedProvider>,
    );

    // ASSERT
    await waitFor(() => {
      expect(
        screen.getByText(`What's In It For My Business?`),
      ).toBeInTheDocument();
    });
    expect(
      screen.getByText(`What's In It For My Business?`).parentElement
        ?.parentElement,
    ).toHaveClass('row:featured-left');
    expect(
      screen.getByText(`What's In It For Me?`).parentElement?.parentElement,
    ).toHaveClass('row:featured-right');
    expect(
      screen.getByText(`What's In It For The Drivers?`).parentElement
        ?.parentElement,
    ).toHaveClass('row:featured-left');
  });

  it('should match snapshot', async () => {
    // ACT
    const getComponent = render(
      <MockedProvider addTypename={false}>
        <FleetLandingPage genericPage={DATA.genericPage} />
      </MockedProvider>,
    );
    // ASSERT
    await waitFor(() => {
      expect(screen.getByText(`Flexible Fleet Management`)).toBeInTheDocument();
    });
    const tree = getComponent.baseElement;
    expect(tree).toMatchSnapshot();
  });
});
