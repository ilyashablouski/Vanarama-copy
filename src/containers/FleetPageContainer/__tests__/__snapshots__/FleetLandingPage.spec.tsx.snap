// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FleetLandingPage /> should match snapshot 1`] = `
<body>
  <div>
    <div
      class="row:bg-hero"
    >
      <div
        class="row:hero"
      >
        <div
          class="hero--left"
        >
          <h1
            class="heading -white -xlarge"
          >
            Flexible Fleet Management 
          </h1>
          <p
            class="text -large -white"
          >
            With 
            <strong>
              No Hidden Costs
            </strong>
          </p>
          <div>
            <div
              class="image hero--image -expand -plain"
            >
              <img
                class="image--native"
                loading="lazy"
                sizes="(min-width:320px) 800px, 1200px"
                src="https://dev.vanarama-nonprod.com/cdn-cgi/image/fit=scale-down,height=620,width=620,quality=59,sharpness=0,metadata=none/https://dev.vanarama-nonprod.com/Assets/images/insurance/home-hero/insurance-hero.png"
                srcset="https://dev.vanarama-nonprod.com/cdn-cgi/image/fit=scale-down,height=620,width=620,quality=59,sharpness=0,metadata=none/https://dev.vanarama-nonprod.com/Assets/images/insurance/home-hero/insurance-hero.png 320w, https://dev.vanarama-nonprod.com/cdn-cgi/image/fit=scale-down,height=620,width=620,quality=59,sharpness=0,metadata=none/https://dev.vanarama-nonprod.com/Assets/images/insurance/home-hero/insurance-hero.png 800w, https://dev.vanarama-nonprod.com/cdn-cgi/image/fit=scale-down,height=620,width=620,quality=59,sharpness=0,metadata=none/https://dev.vanarama-nonprod.com/Assets/images/insurance/home-hero/insurance-hero.png 1200w"
              />
            </div>
          </div>
        </div>
        <div
          class="hero--right"
          style="min-height: 347px;"
        >
          <div
            class="card hero-card"
            data-testid="card"
          >
            <div
              class="hero-card--inner"
            >
              <p
                class="heading -primary -lead"
              >
                Schedule A Call With Us
              </p>
              <form
                class="form"
                data-testid="fleet-request-call-back-form"
                id="fleet-request-call-back-form"
              >
                <div
                  class="formgroup"
                >
                  <label
                    class="formgroup--legend"
                    for="fleet-call-back-form_full-name"
                  >
                    Full Name
                  </label>
                  <div
                    class="formgroup--content"
                  >
                    <div
                      class="textinput regular"
                    >
                      <input
                        class="textinput--native regular"
                        data-testid="fleet-call-back-form_full-name"
                        id="fleet-call-back-form_full-name"
                        name="fullName"
                        type="text"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="formgroup"
                >
                  <label
                    class="formgroup--legend"
                    for="fleet-call-back-form_company-name"
                  >
                    Company Name
                  </label>
                  <div
                    class="formgroup--content"
                  >
                    <div
                      class="textinput regular"
                    >
                      <input
                        class="textinput--native regular"
                        data-testid="fleet-call-back-form_company-name"
                        name="companyName"
                        type="text"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="formgroup"
                >
                  <label
                    class="formgroup--legend"
                    for="fleet-call-back-form_fleet-name"
                  >
                    Fleet Size
                  </label>
                  <div
                    class="formgroup--content"
                  >
                    <div
                      class="textinput regular"
                    >
                      <input
                        class="textinput--native regular"
                        data-testid="fleet-call-back-form_fleet-size"
                        name="fleetSize"
                        type="number"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="formgroup"
                >
                  <label
                    class="formgroup--legend"
                    for="fleet-call-back-form_email-name"
                  >
                    Email Address
                  </label>
                  <div
                    class="formgroup--content"
                  >
                    <div
                      class="textinput regular"
                    >
                      <input
                        class="textinput--native regular"
                        data-testid="fleet-call-back-form_email-address"
                        name="email"
                        type="text"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="formgroup"
                >
                  <label
                    class="formgroup--legend"
                    for="fleet-call-back-form_phone-number"
                  >
                    Phone Number
                  </label>
                  <div
                    class="formgroup--content"
                  >
                    <div
                      class="textinput regular"
                    >
                      <input
                        class="textinput--native regular"
                        data-testid="fleet-call-back-form_phone-number"
                        id="fleet-call-back-form_phone-number"
                        name="phoneNumber"
                        type="text"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="formgroup"
                >
                  <div
                    class="formgroup--content"
                  >
                    <div
                      class="checkbox -teal"
                    >
                      <input
                        class="checkbox--native"
                        data-testid="aboutTermsAndCons"
                        id="termsAndConsregister-form-terms"
                        name="termsAndCons"
                        type="checkbox"
                        value=""
                      />
                      <label
                        class="checkbox--label"
                        for="termsAndConsregister-form-terms"
                      >
                        <span
                          class="text -regular -darker"
                        >
                          I agree to the 
                          <a
                            class="link -teal"
                            href="/legal/terms-and-conditions.html"
                            target="_blank"
                          >
                            Terms and Conditions
                          </a>
                        </span>
                      </label>
                    </div>
                    <div
                      class="checkbox -teal"
                    >
                      <input
                        class="checkbox--native"
                        data-testid="aboutPrivacyPolicy"
                        id="privacyPolicyregister-form-privacy-policy"
                        name="privacyPolicy"
                        type="checkbox"
                        value=""
                      />
                      <label
                        class="checkbox--label"
                        for="privacyPolicyregister-form-privacy-policy"
                      >
                        <span
                          class="text -regular -darker"
                        >
                          I have read and understood the 
                          <a
                            class="link -teal"
                            href="/legal/privacy-policy.html"
                            target="_blank"
                          >
                            Privacy Policy
                          </a>
                        </span>
                      </label>
                    </div>
                    <div
                      class="checkbox -teal"
                    >
                      <input
                        class="checkbox--native"
                        data-testid="aboutConsent"
                        id="consentregister-form-consent"
                        name="consent"
                        type="checkbox"
                        value=""
                      />
                      <label
                        class="checkbox--label"
                        for="consentregister-form-consent"
                      >
                        <span
                          class="text -regular -darker"
                        >
                          Keep me updated on the latest deals & offers
                        </span>
                      </label>
                    </div>
                  </div>
                </div>
                <button
                  class="button -fullwidth -primary -large -solid"
                  data-testid="fleet-call-back-form_continue"
                  type="submit"
                >
                  <div
                    class="button--inner"
                  >
                    I'd Like A Callback
                  </div>
                </button>
              </form>
            </div>
          </div>
          <div
            class="hero-benefits-bar"
          >
            <div
              class="slider"
              style="box-sizing: border-box; display: block; height: inherit; position: relative; width: 100%;"
              tabindex="0"
            >
              <div
                class="slider-frame"
                style="box-sizing: border-box; display: block; height: 100%; margin: 0px; overflow: hidden; padding: 0px; position: relative; transform: translate3d(0, 0, 0);"
              >
                <ul
                  class="slider-list"
                  style="box-sizing: border-box; cursor: pointer; display: block; height: 0px; margin: 0px 0px; padding: 0px; position: relative; transform: translate3d(0px, 0px, 0); width: auto; transition: 0s;"
                >
                  <li
                    class="slider-slide slide-visible slide-current"
                    style="box-sizing: border-box; display: inline-block; height: auto; left: 0px; list-style-type: none; margin: auto 0px auto 0px; position: absolute; top: 0px; transform: scale(1); transition: transform .4s linear; vertical-align: top; width: 0px;"
                    tabindex="-1"
                  >
                    <div
                      aria-hidden="false"
                      tabindex="0"
                    >
                      <div
                        class="text -small -black"
                      >
                        <b>
                          Lowest Price Guaranteed
                        </b>
                      </div>
                      <div
                        class="text -small -darker"
                      >
                        We'll beat any price or give you £100.
                      </div>
                    </div>
                  </li>
                  <li
                    aria-hidden="true"
                    class="slider-slide"
                    inert=""
                    style="box-sizing: border-box; display: inline-block; height: auto; left: 0px; list-style-type: none; margin: auto 0px auto 0px; position: absolute; top: 0px; transform: scale(1); transition: transform .4s linear; vertical-align: top; width: 0px;"
                  >
                    <div
                      aria-hidden="true"
                    >
                      <div
                        class="text -small -black"
                      >
                        <b>
                          Our Customers Love Us
                        </b>
                      </div>
                      <div
                        class="text -small -darker"
                      >
                        <span
                          class="b-bar-rating"
                        >
                          ★★★★★
                        </span>
                         on Trustpilot.
                      </div>
                    </div>
                  </li>
                  <li
                    aria-hidden="true"
                    class="slider-slide"
                    inert=""
                    style="box-sizing: border-box; display: inline-block; height: auto; left: 0px; list-style-type: none; margin: auto 0px auto 0px; position: absolute; top: 0px; transform: scale(1); transition: transform .4s linear; vertical-align: top; width: 0px;"
                  >
                    <div
                      aria-hidden="true"
                    >
                      <div
                        class="text -small -black"
                      >
                        <b>
                          Over 100,000 Vehicles Leased
                        </b>
                      </div>
                      <div
                        class="text -small -darker"
                      >
                        Van, pickup or car - lease yours today.
                      </div>
                    </div>
                  </li>
                </ul>
              </div>
              <div
                class="slider-control-centerleft"
                style="position: absolute; top: 50%; left: 0px; transform: translateY(-50%);"
              >
                <button
                  type="button"
                >
                  <span>
                    Previous
                  </span>
                </button>
              </div>
              <div
                class="slider-control-centerright"
                style="position: absolute; top: 50%; right: 0px; transform: translateY(-50%);"
              >
                <button
                  type="button"
                >
                  <span>
                    Next
                  </span>
                </button>
              </div>
              <div
                class="slider-control-bottomcenter"
                style="position: absolute; bottom: 0px; left: 50%; transform: translateX(-50%);"
              >
                <ul
                  style="position: relative; top: -10px; display: flex; margin: 0px; padding: 0px; list-style-type: none;"
                >
                  <li
                    class="paging-item active"
                  >
                    <button
                      aria-label="slide 1 bullet"
                      style="cursor: pointer; opacity: 1; background: transparent; fill: black;"
                      type="button"
                    >
                      <svg
                        class="paging-dot"
                        height="6"
                        width="6"
                      >
                        <circle
                          cx="3"
                          cy="3"
                          r="3"
                        />
                      </svg>
                    </button>
                  </li>
                  <li
                    class="paging-item"
                  >
                    <button
                      aria-label="slide 2 bullet"
                      style="cursor: pointer; opacity: 0.5; background: transparent; fill: black;"
                      type="button"
                    >
                      <svg
                        class="paging-dot"
                        height="6"
                        width="6"
                      >
                        <circle
                          cx="3"
                          cy="3"
                          r="3"
                        />
                      </svg>
                    </button>
                  </li>
                  <li
                    class="paging-item"
                  >
                    <button
                      aria-label="slide 3 bullet"
                      style="cursor: pointer; opacity: 0.5; background: transparent; fill: black;"
                      type="button"
                    >
                      <svg
                        class="paging-dot"
                        height="6"
                        width="6"
                      >
                        <circle
                          cx="3"
                          cy="3"
                          r="3"
                        />
                      </svg>
                    </button>
                  </li>
                </ul>
              </div>
              <style
                type="text/css"
              >
                .slider-slide &gt; img { width: 100%; display: block; }
          .slider-slide &gt; img:focus { margin: auto; }
              </style>
            </div>
          </div>
        </div>
        <div
          class="hero--decals"
        >
          <svg
            class="hero--curve"
            data-name="curve"
            id="hero--curve"
            viewBox="0 0 1920 170.2"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="hero--curve-background"
              d="M0,65.16v107H1920V113.35s-271.62,32.89-625.91,42.9C925.77,179.08,394.27,183,0,65.16Z"
              transform="translate(0 -2)"
            />
            <path
              class="hero--curve-foreground"
              d="M0,2V64.75c394.27,117.82,925.77,113.92,1294.09,91.08C874,167.71,337.57,147.42,0,2Z"
              transform="translate(0 -1)"
            />
          </svg>
          <svg
            class="hero--curve-m"
            data-name="curve"
            id="hero--curve-m"
            viewBox="0 0 375 65.05"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="hero--curve-foreground"
              d="M0,1.81v35c122.83,16.41,249.92,25.7,375,30V47.6C246.33,38.57,119.07,24,0,1.81Z"
              transform="translate(0 -1)"
            />
            <path
              class="hero--curve-background"
              d="M0,66.86H375c-125.08-4.32-252.17-13.61-375-30Z"
              transform="translate(0 -1.81)"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      class="row:lead-text"
    >
      <p
        class="heading -black -xlarge"
      >
        Why You Should Trust Your Fleet With Vanarama
      </p>
      <span
        class="text -lead -darker"
      >
        The prospect of managing a mixed fleet of multiple vehicles, on varied financial contracts, is daunting...but we'll take that away. Our dedicated team, and the platforms available to you and us, mean that running your mixed fleet is hassle-free
      </span>
    </div>
    <div
      class="row:featured-right"
    >
      <div
        class="card -overflow"
        data-testid="card-review"
      >
        <span
          class="text review-text -regular -dark"
          data-testid="review"
        >
          Vanarama offered the cheapest prices on the vehicles and their service is fully integrated from telephone support to a state-of-the-art app, for Drivers and Fleet Managers.
        </span>
        <div
          class="review-author"
          data-testid="author-review"
        >
          Peter Cheshire
        </div>
        <div
          class="review-decoration"
        >
          <div
            class="rating -regular -orange"
          >
            <span>
              <i
                class="icon -orange"
              >
                <svg
                  height="1em"
                  viewBox="0 0 512 512"
                  width="1em"
                >
                  <path
                    d="M496,203.3H312.36L256,32,199.64,203.3H16L166.21,308.7,107.71,480,256,373.84,404.29,480,345.68,308.7Z"
                  />
                </svg>
              </i>
            </span>
            <span>
              <i
                class="icon -orange"
              >
                <svg
                  height="1em"
                  viewBox="0 0 512 512"
                  width="1em"
                >
                  <path
                    d="M496,203.3H312.36L256,32,199.64,203.3H16L166.21,308.7,107.71,480,256,373.84,404.29,480,345.68,308.7Z"
                  />
                </svg>
              </i>
            </span>
            <span>
              <i
                class="icon -orange"
              >
                <svg
                  height="1em"
                  viewBox="0 0 512 512"
                  width="1em"
                >
                  <path
                    d="M496,203.3H312.36L256,32,199.64,203.3H16L166.21,308.7,107.71,480,256,373.84,404.29,480,345.68,308.7Z"
                  />
                </svg>
              </i>
            </span>
            <span>
              <i
                class="icon -orange"
              >
                <svg
                  height="1em"
                  viewBox="0 0 512 512"
                  width="1em"
                >
                  <path
                    d="M496,203.3H312.36L256,32,199.64,203.3H16L166.21,308.7,107.71,480,256,373.84,404.29,480,345.68,308.7Z"
                  />
                </svg>
              </i>
            </span>
            <span>
              <i
                class="icon -orange"
              >
                <svg
                  height="1em"
                  viewBox="0 0 512 512"
                  width="1em"
                >
                  <path
                    d="M496,203.3H312.36L256,32,199.64,203.3H16L166.21,308.7,107.71,480,256,373.84,404.29,480,345.68,308.7Z"
                  />
                </svg>
              </i>
            </span>
            <span
              class="text -regular -dark"
            >
              5
            </span>
          </div>
        </div>
      </div>
      <div>
        <p
          class="heading -black -large"
        >
          Wherever You Are, However You Want To Work, We're There For You
        </p>
        <p
          class="text -regular -darker"
        >
          The prospect of managing a mixed fleet of multiple vehicles, on varied financial contracts, is daunting...but we'll take that away. Our dedicated team, and the platforms available to you and us, mean that running your mixed fleet is hassle-free.
        </p>
        <button
          class="button -primary -regular -solid"
          data-testid="fleet_testimonial-section_request-button"
        >
          <div
            class="button--inner"
          >
            I'd Like A Callback
          </div>
        </button>
      </div>
    </div>
    <div
      class="row:featured-left"
    >
      <div>
        <p
          class="heading -black -large"
        >
          What's In It For My Business?
        </p>
        <h3
          class="heading -primary -regular"
        >
          Every fleet is different:
        </h3>
        <p
          class="text -regular -darker"
        >
          Your fleet needs are unique and that's why entrusting yours to a company you trust is important.
        </p>
        <h3
          class="heading -primary -regular"
        >
          Best price for your business:
        </h3>
        <p
          class="text -regular -darker"
        >
          Choosing a fleet management company that works with a network of funders means they can provide the very best prices on the market first time.
        </p>
        <h3
          class="heading -primary -regular"
        >
          Compliance as standard:
        </h3>
        <p
          class="text -regular -darker"
        >
          Duty of care is a key concern and goes hand-in-hand with staying compliant. Driver risk assessments and safety checks need to be carried out on time and to high standards, so work with a company that understands how to keep you organised.
        </p>
      </div>
    </div>
    <div
      class="row:featured-right"
    >
      <div>
        <p
          class="heading -black -large"
        >
          What's In It For Me?
        </p>
        <h3
          class="heading -primary -regular"
        >
          Hassle-free fleet reporting:
        </h3>
        <p
          class="text -regular -darker"
        >
          From MOT notifications to duty of care compliance tasks, keeping information at your fingertips is vital to effective fleet management.
        </p>
        <h3
          class="heading -primary -regular"
        >
          Protect your fleet:
        </h3>
        <p
          class="text -regular -darker"
        >
          Having your fleet managed and insured through one channel makes life less complicated for you.
        </p>
        <h3
          class="heading -primary -regular"
        >
          Cost-effective fleet maintenance:
        </h3>
        <p
          class="text -regular -darker"
        >
          Easy access to all routine servicing and repairs, tyre replacement, MOTs and vehicle breakdown cover is true peace-of-mind.
        </p>
        <h3
          class="heading -primary -regular"
        >
          Sale and lease back:
        </h3>
        <p
          class="text -regular -darker"
        >
          Removing vehicles from outright purchase and into a lease agreement allows you to grow your business faster by giving you the option to invest cash into your business rather than the vehicles.
        </p>
        <h3
          class="heading -primary -regular"
        >
          Controlling costs:
        </h3>
        <p
          class="text -regular -darker"
        >
          A robust reporting platform will allow you to look at maintenance budgets, total driver costs and whole-life fleet costings at a moment's notice.
        </p>
      </div>
      <div
        class="image -expand"
      >
        <img
          alt="AudiQ30718 4 k5ojqt"
          class="image--native"
          loading="eager"
          sizes="(min-width:320px) 800px, 1200px"
          src="https://dev.vanarama-nonprod.com/cdn-cgi/image/fit=scale-down,height=1920,width=1920,quality=75,sharpness=0,metadata=none/https://images.ctfassets.net/3xid768u5joa/2QbzKD75ObIHIeVMzNeCF/1455166780ba4eb095c965e278a63b69/AudiQ30718_4_k5ojqt.jpg"
          srcset="https://dev.vanarama-nonprod.com/cdn-cgi/image/fit=scale-down,height=320,width=320,quality=60,sharpness=0,metadata=none/https://images.ctfassets.net/3xid768u5joa/2QbzKD75ObIHIeVMzNeCF/1455166780ba4eb095c965e278a63b69/AudiQ30718_4_k5ojqt.jpg 320w, https://dev.vanarama-nonprod.com/cdn-cgi/image/fit=scale-down,height=800,width=800,quality=70,sharpness=0,metadata=none/https://images.ctfassets.net/3xid768u5joa/2QbzKD75ObIHIeVMzNeCF/1455166780ba4eb095c965e278a63b69/AudiQ30718_4_k5ojqt.jpg 800w, https://dev.vanarama-nonprod.com/cdn-cgi/image/fit=scale-down,height=1200,width=1200,quality=75,sharpness=0,metadata=none/https://images.ctfassets.net/3xid768u5joa/2QbzKD75ObIHIeVMzNeCF/1455166780ba4eb095c965e278a63b69/AudiQ30718_4_k5ojqt.jpg 1200w"
        />
      </div>
    </div>
    <div
      class="row:featured-left"
    >
      <div>
        <p
          class="heading -black -large"
        >
          What's In It For The Drivers?
        </p>
        <h3
          class="heading -primary -regular"
        >
          Maintenance when it's needed:
        </h3>
        <p
          class="text -regular -darker"
        >
          No matter which maintenance package you choose, the drivers themselves can arrange bookings through the Driverhelpline application or by calling the support team.
        </p>
        <h3
          class="heading -primary -regular"
        >
          Comprehensive vehicle management:
        </h3>
        <p
          class="text -regular -darker"
        >
          How great would it be if drivers knew that support was just a tap on an app away? They'll benefit from the same up-to-date reporting and service-on-demand functions as you. Drivers know they're being compliant, and you can rest easy.
        </p>
        <h3
          class="heading -primary -regular"
        >
          Peace of mind and support:
        </h3>
        <p
          class="text -regular -darker"
        >
          For drivers, the fleet management relationship shouldn't be centred on vehicle delivery, they want support. And they'll have questions they need answered. That's when having a dedicated app filled with the information they need, and a dedicated support team really will pay for itself.
        </p>
      </div>
    </div>
    <hr
      class="-fullwidth"
    />
    <div
      class="row:features-4col"
    >
      <p
        class="heading -black -lead"
      >
        Not Quite Convinced? Also Included:
      </p>
      <div
        class="tile -button -plain tile--centered"
      >
        <span />
        <span
          class="tile--link"
        >
          <p
            class="heading -black -regular"
          >
            Price Protection
          </p>
        </span>
        <p
          class="text -regular -darker"
        >
          Pretium facilisi etiam pretium, cras interdum enim, nullam.
        </p>
      </div>
      <div
        class="tile -button -plain tile--centered"
      >
        <span />
        <span
          class="tile--link"
        >
          <p
            class="heading -black -regular"
          >
            Customer Reviews
          </p>
        </span>
        <p
          class="text -regular -darker"
        >
          Pretium facilisi etiam pretium, cras interdum enim, nullam.
        </p>
      </div>
      <div
        class="tile -button -plain tile--centered"
      >
        <span />
        <span
          class="tile--link"
        >
          <p
            class="heading -black -regular"
          >
            Quote Online
          </p>
        </span>
        <p
          class="text -regular -darker"
        >
          Pretium facilisi etiam pretium, cras interdum enim, nullam.
        </p>
      </div>
      <div
        class="tile -button -plain tile--centered"
      >
        <span />
        <span
          class="tile--link"
        >
          <p
            class="heading -black -regular"
          >
            Confused About Leasing?
          </p>
        </span>
        <p
          class="text -regular -darker"
        >
          Pretium facilisi etiam pretium, cras interdum enim, nullam.
        </p>
      </div>
    </div>
  </div>
</body>
`;
